service.go

```diff
@@ -77,15 +79,15 @@ type ListenerConfig struct {
 	Measures Measures
 }
 
-type service struct {
+type ClientService struct {
```

Comment:
renamed to ClientService in order to be able to export in caduceus (https://github.com/xmidt-org/caduceus/pull/526/files#diff-222abc748c3cc33bd3a92a4ade6ad28f46a839351844d019cc6e7c3228fef941R16)